spec.html

```diff
@@ -212,6 +212,135 @@ <h1>
   </emu-clause>
 </emu-clause>
 
+<emu-clause id="sec-ordinary-and-exotic-objects-behaviours">
+  <h1>Ordinary and Exotic Objects Behaviours</h1>
+  <emu-clause id="sec-built-in-exotic-object-internal-methods-and-slots">
+    <h1>Built-in Exotic Object Internal Methods and Slots</h1>
```

Comment:
This exotic object language is unnecessarily complicated. See https://tc39.es/ecma262/#sec-promise-resolve-functions for a much simpler example of how to create JS-level functions that close over some things.

Reply:
Done.